spec/ParseGeoPoint.spec.js

```diff
@@ -628,4 +738,60 @@ describe('Parse.GeoPoint testing', () => {
       done();
     });
   });
+
+  it('support $geoIntersects queries', (done) => {
```

Comment:
This test already exists

https://github.com/parse-community/parse-server/blob/master/spec/ParsePolygon.spec.js#L130